compiler/rustc_mir/src/dataflow/framework/graphviz.rs

```diff
@@ -578,7 +578,7 @@ where
         return String::new();
     }
 
-    let re = Regex::new("\u{001f}([+-])").unwrap();
+    let re = Regex::new("\t?\u{001f}([+-])").unwrap();
```

Comment:
Why not using OnceCell here for the Regex to avoid repeatedly compiling ?

Reply:
This is debug-only code so it doesn't matter too much in practice, and I'd guess this regex is so small it compiles nearly instantaneously. I wouldn't object to a PR adding a OnceCell if it didn't add too much noise to this code.